appveyor.yml

```diff
@@ -126,8 +126,9 @@ before_deploy:
   - "SET PATH=%ORIGPATH%"
   - "%CMD_IN_ENV% %WP_INSTDIR%/scripts/env.bat"
   - "%CMD_IN_ENV% python.exe setup.py install"
-  - "%CMD_IN_ENV% easy_install --upgrade pip"
```

Comment:
Maybe upgrade with ` - "%CMD_IN_ENV% python -m pip install --upgrade pip` since we now patch packages?